composer.json

```diff
@@ -25,6 +25,7 @@
         "php": "^7.2",
         "doctrine/common": "^2.7",
         "doctrine/inflector": "^1.4 || ^2.0",
+        "doctrine/persistence": "^1.3.3",
```

Comment:
This should go to the `require-dev` section, we don't refer to this in `src`. If one our users has the issue, they have it because of `symfony/security-acl`, not because of us.

Reply:
Actually we do refer to persistence classes in `src/Command/GenerateObjectAclCommand.php`, so maybe this is fine…

Reply:
What should I do than?

Reply:
I think leave it as is, it's not a huge deal…